lib/modules/platform/gitea/index.spec.ts

```diff
@@ -177,6 +181,8 @@ describe('modules/platform/gitea/index', () => {
     gitvcs = require('../../../util/git');
     gitvcs.isBranchStale.mockResolvedValue(false);
     gitvcs.getBranchCommit.mockReturnValue(mockCommitHash);
+    hostRules = (await import('../../../util/host-rules')) as any;
```

Comment:
please use static important

Reply:
What exactly do you mean by static import? Using require instead of import (`hostRules = require('../../../util/host-rules');`) or removing the `await` command / `as any` instruction? Sorry I'm not so familiar with import statements.

Reply:
move to top of file and use `import ... from ...;` syntax

Reply:
If we use the suggested import, there are two different hostRules instances because `jest.resetModules()` is executed before every test ([see docu](https://jestjs.io/docs/jest-object#jestresetmodules)). Thus, it's necessary to import hostRules in the beforeEach block or in of the corresponding test after reseting the modules.

Reply:
ok, right 😕 So at lease some more type safty

```suggestion
    hostRules = mocked(await import('../../../util/host-rules'));
```

Please change the other dynamic imports accordingly